lib/rpcServer/schemas/addresses.json

```diff
@@ -2,7 +2,11 @@
   "type": "object",
   "properties": {
     "address": {
-      "type": ["array", "string"]
+      "type": ["array", "string"],
+      "oneOf": [
+        { "type": "string",  "maxLength": 34, "minLength": 26},
```

Comment:
I'm not really sure why address is from 26 to 34 characters long. Address size is 160 bit + 8 bit version prefix, which results in 21 byte. 21 byte as a hex string contains 42 characters, base58 string contains  34 characters. Where 26 comes from?

Reply:
On a different note: we should review those validations, as they right now just a placeholders mostly. But this is a subject to another task.

Reply:
https://en.bitcoin.it/wiki/Technical_background_of_version_1_Bitcoin_addresses
Mainline addresses can be 25-34 characters in length, and testnet addresses can be 26-34 characters in length. Most addresses are 33 or 34 characters long.

https://bitcoin.stackexchange.com/questions/36944/what-are-the-minimum-and-maximum-lengths-of-a-mainnet-bitcoin-address

Reply:
@antouhou fyi: here is a thread were bitcoin address validation is discussed. Among other things they are discussing a vanity address with 27 characters:
https://bitcointalk.org/index.php?topic=1026.0
Not sure if the exact same rules apply for dash, but it is a reasonable expectation

Reply:
dash has the same logic as bitcoin for verification and it has not been changed since that time https://github.com/dashpay/dash/blame/master/src/base58.cpp#L252